sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -98,7 +94,11 @@ def with_prefix(self, prefix):
 
 
 class _CombiningValueStateTag(_StateTag):
-  """StateTag pointing to an element, accumulated with a combiner."""
+  """StateTag pointing to an element, accumulated with a combiner.
+
+  The given tag must be unique for this stage.  If CombineFn is None then
```

Comment:
What is a stage? This is not a concept in the Beam model. The tag must be unique for a step / transform / DoFn.